### PR TITLE
Add documentation on import delete profiles CSV

### PR DESCRIPTION
**Pull request by anton-ko**
re: https://trello.com/c/OdzvpcPw/3835-bulk-tagging-and-deleting-for-users

Adds documentation for the new functionality

### CODE REVIEW

**Comment by anton-ko** [comment]
[Bulk tagging and deleting for users](https://trello.com/c/OdzvpcPw/3835-bulk-tagging-and-deleting-for-users)

File: docs/apis/imports.md [hunk 1]
```diff
@@ -342,6 +344,95 @@ Response:
 }
 ```
 
+##### Using `kind=delete_csv` to tag User Profiles via a User ID :id=examples-profiles-delete-by-id
```
Comment (bnorton) [inline docs/apis/imports.md:347]:
```suggestion
##### Using `kind=delete_csv` to delete User Profiles via a User ID :id=examples-profiles-delete-by-id
```

File: docs/apis/imports.md [hunk 2]
```diff
@@ -342,6 +344,95 @@ Response:
...
+5a1fe53
+621f8e7
+```
+
+Request:
+
+```json
+{
+  "kind": "delete_csv",
+  "name": "Feedback Request: Post-BETA1",
```
Comment (bnorton) [inline docs/apis/imports.md:364]:
```suggestion
  "name": "Data deletion request 54Dw",
```

File: docs/apis/imports.md [hunk 3]
```diff
@@ -342,6 +344,95 @@ Response:
...
+jill@sample.com
+aaron@example.com
+```
+
+Request:
+
+```json
+{
+  "kind": "delete_csv",
+  "name": "Feedback Request: Post-BETA2",
```
Comment (bnorton) [inline docs/apis/imports.md:408]:
```suggestion
  "name": "Data deletion request 65fx",
```

File: docs/apis/imports.md [hunk 4]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
 
+<details>
+<summary>Tagging User Profiles using Filters</summary>
+
+- `kind=tag_filters` + `model_kind=profile` means Tag Users Profiles matching the given Filters.
+
+```bash
+curl -X POST -H 'X-Account-Secret: ACCOUNT_SECRET' \
+  -H 'Content-Type: application/json' \
+  -d '{"name": "Gmail users", "model_kind": "profile", "kind": "tag_filters", "on_model_missing": "create", "filters": [{"kind": "property", "prop": "email", "op": "in", "value": "gmail.com"}], "import_at": "$now"}' \
```
Comment (bnorton) [inline docs/apis/imports.md:659]:
```suggestion
  -d '{"name": "Gmail users", "model_kind": "profile", "kind": "tag_filters", "filters": [{"kind": "property", "prop": "email", "op": "in", "value": "gmail.com"}], "import_at": "$now"}' \
```

File: docs/apis/imports.md [hunk 5]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+```
+
+</details>
+
+<details>
+<summary>Deleting User Profiles by <b>UID</b></summary>
+
+- `kind=delete_csv` + `model_kind=profile` means Delete Users Profiles by CSV.
+
+With a CSV like this (`feedback-request-post-BETA1.csv`):
```
Comment (bnorton) [inline docs/apis/imports.md:676]:
```suggestion
With a CSV like this (`data-deletion-request-54Dw.csv`):
```

File: docs/apis/imports.md [hunk 6]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+2de2c71
+632665
+```
+
+First, Create the import, naming it and mapping the `User ID` CSV header to the `uid` Chameleon property:
+
+```bash
+curl -X POST -H 'X-Account-Secret: ACCOUNT_SECRET' \
+  -H 'Content-Type: application/json' \
+  -d '{"name": "Feedback Request: Post-BETA1", "model_kind": "profile", "kind": "delete_csv", "on_model_missing": "create", "properties": [{"name":"User ID","prop":"uid"}] }' \
```
Comment (bnorton) [inline docs/apis/imports.md:690]:
```suggestion
  -d '{"name": "Data deletion request 54Dw", "model_kind": "profile", "kind": "delete_csv", "properties": [{"name":"User ID","prop":"uid"}] }' \
```

File: docs/apis/imports.md [hunk 7]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+  'https://api.trychameleon.com/edit/v3/imports'
+```
+
+Then Upload the CSV called `feedback-request-post-BETA1.csv` and trigger the import with `import_at=now`
```
Comment (bnorton) [inline docs/apis/imports.md:694]:
```suggestion
Then Upload the CSV called `data-deletion-request-54Dw.csv` and trigger the import with `import_at=now`
```

File: docs/apis/imports.md [hunk 8]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+
+- Use the `import.id` from the last request in place of IMPORT_ID:
+
+```bash
+curl -X PATCH -H 'X-Account-Secret: ACCOUNT_SECRET' \
+  -F file=@feedback-request-post-BETA1.csv \
```
Comment (bnorton) [inline docs/apis/imports.md:700]:
```suggestion
  -F file=@data-deletion-request-54Dw.csv \
```

File: docs/apis/imports.md [hunk 9]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+
+</details>
+
 
+<details>
+<summary>Deleting User Profiles by <b>Email</b></summary>
 
+- `kind=delete_csv` + `model_kind=profile` means Delete Users Profiles by CSV.
 
+With a CSV like this (`feedback-request-post-BETA1.csv`):
```
Comment (bnorton) [inline docs/apis/imports.md:718]:
```suggestion
With a CSV like this (`data-deletion-request-54Dw.csv`):
```

File: docs/apis/imports.md [hunk 10]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+  'https://api.trychameleon.com/edit/v3/imports'
+```
+
+Then Upload the CSV called `feedback-request-post-BETA1.csv` and trigger the import with `import_at=now`
+
+- Use the `import.id` from the last request in place of IMPORT_ID:
+
+```bash
+curl -X PATCH -H 'X-Account-Secret: ACCOUNT_SECRET' \
+  -F file=@feedback-request-post-BETA1.csv \
```
Comment (bnorton) [inline docs/apis/imports.md:742]:
```suggestion
  -F file=@data-deletion-request-54Dw.csv \
```

File: docs/apis/imports.md [hunk 11]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+First, Create the import, naming it and mapping the `Email address` CSV header to the `email` Chameleon property:
+
+```bash
+curl -X POST -H 'X-Account-Secret: ACCOUNT_SECRET' \
+  -H 'Content-Type: application/json' \
+  -d '{"name": "Feedback Request: Post-BETA1", "model_kind": "profile", "kind": "delete_csv", "properties": [{"name":"Email address","prop":"email"}] }' \
+  'https://api.trychameleon.com/edit/v3/imports'
+```
+
+Then Upload the CSV called `feedback-request-post-BETA1.csv` and trigger the import with `import_at=now`
```
Comment (bnorton) [inline docs/apis/imports.md:736]:
```suggestion
Then Upload the CSV called `data-deletion-request-54Dw.csv` and trigger the import with `import_at=now`
```

File: docs/apis/imports.md [hunk 12]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+
+<details>
+<summary>Deleting User Profiles using Filters</summary>
+
+- `kind=delete_filters` + `model_kind=profile` means Delete Users Profiles matching the given Filters.
+
+```bash
+curl -X POST -H 'X-Account-Secret: ACCOUNT_SECRET' \
+  -H 'Content-Type: application/json' \
+  -d '{"name": "Remove users not seen in a year", "model_kind": "profile", "kind": "tag_filters", "on_model_missing": "create", "filters": [{"kind": "property","prop":"last_seen_at","op":"lt-d","value":"365"}], "import_at": "$now"}' \
```
Comment (bnorton) [inline docs/apis/imports.md:762]:
```suggestion
  -d '{"name": "Remove users not seen in a year", "model_kind": "profile", "kind": "tag_filters", "filters": [{"kind": "property","prop":"last_seen_at","op":"lt-d","value":"365"}], "import_at": "$now"}' \
```

**Review by bnorton — APPROVED** [review]

File: docs/apis/imports.md [hunk 13]
```diff
@@ -557,11 +648,128 @@ curl -H 'X-Account-Secret: ACCOUNT_SECRET' 'https://api.trychameleon.com/edit/v3
...
+
+<details>
+<summary>Deleting User Profiles using Filters</summary>
+
+- `kind=delete_filters` + `model_kind=profile` means Delete Users Profiles matching the given Filters.
+
+```bash
+curl -X POST -H 'X-Account-Secret: ACCOUNT_SECRET' \
+  -H 'Content-Type: application/json' \
+  -d '{"name": "Remove users not seen in a year", "model_kind": "profile", "kind": "tag_filters", "filters": [{"kind": "property","prop":"last_seen_at","op":"lt-d","value":"365"}], "import_at": "$now"}' \
```
Comment (anton-ko) [inline docs/apis/imports.md:762]:
```suggestion
  -d '{"name": "Remove users not seen in a year", "model_kind": "profile", "kind": "delete_filters", "filters": [{"kind": "property","prop":"last_seen_at","op":"lt-d","value":"365"}], "import_at": "$now"}' \
```

**Pull request merged**